buildSrc/build.gradle.kts

```diff
@@ -87,6 +87,10 @@ allprojects {
                 excludeModule("com.google.j2objc", "j2objc-annotations")
             }
         }
+        maven {
+            name = "Gradle Enterprise Gradle plugin RC"
+            url = uri("https://repo.gradle.org/gradle/enterprise-libs-release-candidates-local")
```

Comment:
Do we need to add this repository to the list of mirrored repositories as well?

Reply:
Added the mirror for it.